hydromt_wflow/wflow.py

```diff
@@ -5116,6 +5110,9 @@ def set_grid(
             Name of new map layer, this is used to overwrite the name of a DataArray and
             ignored if data is a Dataset
         """
+        if "idx_out" in data.coords:
+            data = data.rename({"idx_out": "meta_subgrid_outlet_idx"})
+
```

Comment:
I see this one is a coords rather than data_vars so that's why it's a bit skipped... I prefer to rename in setup_basemaps where it is created rather than in set_grid function